addons/project/i18n/project.pot

```diff
@@ -6,8 +6,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 13.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-04-27 09:21+0000\n"
-"PO-Revision-Date: 2020-04-27 09:21+0000\n"
+"POT-Creation-Date: 2022-06-03 14:56+0000\n"
+"PO-Revision-Date: 2022-06-03 14:56+0000\n"
```

Comment:
I am unsure we really need to update those fields... I have just asked @mart-e 
Let's wait for his answer 🙂

Reply:
I have the confirmation that this diff can be removed 🙂